{{ cookiecutter.project_slug }}/backend/Dockerfile

```diff
@@ -10,5 +10,8 @@ LABEL maintainer="{{ cookiecutter.author }} <{{ cookiecutter.email }}>" \
 COPY . .
 
 # Install local requirements and fix permissions
-RUN ./bin/pip install -r requirements.txt \
+RUN ./bin/pip install mxdev \
+    && mv requirements-docker.txt requirements.txt \
+    && ./bin/mxdev -c mx.ini \
+    && ./bin/pip install -r requirements-mxdev.txt \
     && chown -R plone: /app
```

Comment:
fwiw, yesterday we discovered a nice alternative syntax for multiline commands: https://docs.docker.com/engine/reference/builder/#here-documents

Reply:
Maybe in a new PR